configs/db_demo.yaml

```diff
@@ -148,7 +148,21 @@ sources:
       sub: skyportal/tests/data/16fil_sub.png
     token: "load_demo_data token"
 taxonomies:
-  - name: Demo taxonomy
+  - name: Simple Taxonomy
+    hierarchy:
+      class: Time-domain Source
```

Comment:
Time-Domain source renders as a class, not a subclass, is this intended? 
<img width="537" alt="Screen Shot 2020-07-23 at 8 22 33 PM" src="https://user-images.githubusercontent.com/2769632/88358038-4fd28400-cd22-11ea-91ce-ecfc7ec47d0e.png">

should class probabilities be shown in addition to subclass probabilities? Or maybe some kind of tree visualization?

Reply:
I think that the tree viz is outside the scope of this PR. It's a major amount of work as you can imagine. We could have a page for taxonomy viz/adding.

Reply:
Removed "Time-domain Source" as a viable class (it's needed by the schema validator for the taxonomy itself)

Reply:
Done.